package.nls.json

```diff
@@ -136,7 +135,7 @@
     "AWS.samcli.userChoice.browse": "Locate SAM CLI...",
     "AWS.samcli.userChoice.visit.install.url": "Get SAM CLI",
     "AWS.samcli.userChoice.update.awstoolkit.url": "Update AWS Toolkit",
-    "AWS.samcli.initWizard.location.prompt": "Select a location for your new project",
+    "AWS.samcli.initWizard.location.prompt": "Select a workspace folder for your new project",
```

Comment:
Do you think that this will make the 'Browse...' option confusing?

Reply:
Hmm, it might. I was thinking that there wasn't much context for the folders that were currently there (since we didn't include any pathing). The alternative is to rename "Browse..." to something like "Choose a Different Folder..."

Reply:
What about the following?

> Select a workspace folder for your new project, or specify a new location

Reply:
I like @bryceitoc9 suggestion to rename Browse more than what I suggested

Reply:
Updated to "Choose a different folder..."